packages/sdk/src/messageExtension/executeWithSSO.ts

```diff
@@ -126,3 +225,44 @@ export async function handleMessageExtensionQueryWithToken(
   }
   return await executionWithToken(context, config ?? {}, scopes, logic);
 }
+
+/**
+ * Users execute query in message extension with SSO or access token.
+ *
+ * @param {TurnContext} context - The context object for the current turn.
+ * @param {OnBehalfOfCredentialAuthConfig} config - User custom the message extension authentication configuration.
+ * @param {initiateLoginEndpoint} initiateLoginEndpoint - Login page for Teams to redirect to.
+ * @param {string| string[]} scopes - The list of scopes for which the token will have access.
+ * @param {function} logic - Business logic when executing the query in message extension with SSO or access token.
+ *
+ * @throws {@link ErrorCode|InternalError} when User invoke not response to message extension query.
+ * @throws {@link ErrorCode|InternalError} when failed to get access token with unknown error.
+ * @throws {@link ErrorCode|TokenExpiredError} when SSO token has already expired.
+ * @throws {@link ErrorCode|ServiceError} when failed to get access token from simple auth server.
+ * @throws {@link ErrorCode|InvalidParameter} when scopes is not a valid string or string array.
+ * @throws {@link ErrorCode|RuntimeNotSupported} when runtime is nodeJS.
+ *
+ * @returns A MessageExtension Response for the activity. If the logic not return any, return void instead.
+ */
+export async function handleMessageExtensionQueryWithSSO(
```

Comment:
why not overload handleMessageExtensionQueryWithToken

Reply:
In typescript, overload is done by optional parameters, which will introduce redundant logic to check parameter count, parameter types and so on, and is not easy for future maintain. So as discussed with PM, we're using new function name for it, and in the future 2.0 release, we will deprecate `handleMessageExtensionQuery` function, and after split sdk, will remove this function.

Reply:
why not deperecate directly

Reply:
Due to docs/template/samples are not fully ready, user may see some deprecate notice but cannot find docs about how to use it, so we plan to add the notice for the next release.

Reply:
ok